src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -1550,9 +1573,9 @@ protected override void HandleVariablesRequestAsync(IRequestResponder<VariablesA
             Object container;
             if (m_variableManager.TryGet(reference, out container))
             {
-                if (container is IDebugStackFrame2)
+                if (container is VariablesRef)
```

Comment:
>container is VariablesRef [](start = 20, length = 25)

nit: should be `container is VariablesRef variablesRef` and then replace the `container as VariablesRef` with `variablesRef`.